ui/pages/routes/routes.component.test.js

```diff
@@ -92,6 +92,73 @@ describe('Routes Component', () => {
   });
 
   describe('render during send flow', () => {
+    it('should render with network change disabled while adding recipient for send flow', async () => {
```

Comment:
Most of these changes are not in https://github.com/MetaMask/metamask-extension/pull/25426 or https://github.com/MetaMask/metamask-extension/pull/25426 and so most of these should not be here

Reply:
I agree that the code difference should highlight the changes, but I am unsure how to do so through IDE used(Visual Studio Code).
<img width="1912" alt="Screenshot 2024-06-26 at 2 24 10 PM" src="https://github.com/MetaMask/metamask-extension/assets/153644847/0b67347d-2a5c-41c8-972b-a11e29f1a433">
<img width="956" alt="Screenshot 2024-06-26 at 2 24 54 PM" src="https://github.com/MetaMask/metamask-extension/assets/153644847/8b7409cb-1da2-4f30-93e6-ea3580323df5">